inductiva/resources/machine_groups.py

```diff
@@ -6,6 +6,34 @@
 from inductiva import resources
 
 
+def estimate_machine_cost(machine_type: str,
```

Comment:
what if you add a parameter "num_machines" here, because then it will make the code in other scripts shorter, there was one moment where you first looked up if the cost was already estimated, I am not sure if that's the right approach, because the user could have changed the machine type along the way, so it might be better to always retrieve the cost according to the latest info.

Reply:
However, note that the Elastic Machine Group uses the single machine price to compute the lower and upper bound. So you would need to call the API twice to compute that. I guess that is the main reason to keep this structure.

Reply:
> For your second part, I think that is not true, since that is on register moment, so if people change stuff I think it will not be registered again, or will it?

it will still register it.

Reply:
Actually, if you change any variable after initialising the class and then try to start the machines this will crash.
Which can make sense in some way, but then, the start() method seems to be relatively pointless.